### DISCUSSION TITLE
SourcesList Error - Proxmox VE Post Install

### DISCUSSION THREAD

**Discussion by ianalexis** [tteck/Proxmox#2620] [Q&A] [answered]
Im getting:
`N: Repository 'Debian bookworm' changed its 'firmware component' value from 'non-free' to 'non-free-firmware'`
`N: More information about this can be found online in the Release notes at: https://www.debian.org/releases/bookworm/amd64/release-notes/ch-information.html#non-free-split`

I think is related with this SourcesList
`deb http://security.debian.org/debian-security bookworm-security main contrib`
related with [https://www.debian.org/releases/stable/errata.en.html](url) maybe it should be
`deb https://deb.debian.org/debian bookworm-proposed-updates main contrib non-free-firmware non-free`
or
`deb http://security.debian.org bookworm-security main contrib`

**Comment by tteck** [comment]
This is not an error; it's an informational message. Try  executing `apt update`, the message _shouldn't_ be displayed again.

**Reply by ianalexis** [reply]
![imagen](https://github.com/tteck/Proxmox/assets/12130714/c923c658-824a-4a55-8c1a-4806833e822b)
Nope, it always appears

**Reply by tteck** [reply]
type `exit` and show the new shell.
```
root@changedetection:~# exit
logout

Debian GNU/Linux 12 changedetection tty1

changedetection login: root (automatic login)

Change Detection LXC provided by https://tteck.github.io/Proxmox/

Last login: Fri Mar  8 15:40:17 EST 2024 on tty1
root@changedetection:~# 
```
I don't _believe_ that LXC was created from a script found here.

**Reply by ianalexis** [reply]
Its the Proxmox srv shell. Not an LXC.
The SourcesList was added by the Proxmox VE Post Install script.

**Reply by ianalexis** [reply]
![imagen](https://github.com/tteck/Proxmox/assets/12130714/4344e5ee-4e9c-47b1-a434-865cbd1a745b)

**Reply by tteck** [reply, marked as answer]
Did you follow the instructions found here
https://www.debian.org/releases/bookworm/amd64/release-notes/ch-information.html#non-free-split

**Reply by tteck** [reply]
You should use the GUI to update, but if you don't want those messages, execute the following command
```
echo 'APT::Get::Update::SourceListWarnings::NonFreeFirmware "false";' >/etc/apt/apt.conf.d/no-bookworm-firmware.conf
```

**Reply by tteck** [reply]
That code is included in the updated Proxmox VE Post Install script.

**Reply by ianalexis** [reply]
i had to add this deb https://deb.debian.org/debian bookworm main non-free-firmware as the debian page says.
Thanks tt.
